### ISSUE TITLE
Perifoveal hypoautofluorescence

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7540]
**Preferred term label:**
Perifoveal hypoautofluorescence

**Synonyms**
Hypofluorescent lesion in the perifoveal region

**Definition (free text, please give PubMed ID)**
Decreased amount of autofluorescence (emission of fluorescent light from ocular structures in the absence of sodium fluorescein) in the retina in the region surrounding the fovea.
Comment: Fundus autofluorescence utilizes a scanning laser ophthalmoscope to stimulate intrinsically autofluorescent molecules of lipofuscin to visualize the retinal pigment epithelium.
PMID: 34800980
**Parent term (use hpo.jax.org/app)**
Hypoautofluorescent retinal lesion HP:0025159

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 300029 
PMID: 34800980  4/4
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
fundus autofluorescence (FAF) imaging. https://pubmed.ncbi.nlm.nih.gov/34410191/

**Comment by pnrobinson** [comment]
added
